### DISCUSSION TITLE
Every WA Dark After Install

### DISCUSSION THREAD

**Discussion by Christhammer** [SFX-WoW/Masque#172] [Issues] [answered]
![dark](https://user-images.githubusercontent.com/31971736/30455255-82e3be68-999f-11e7-826f-33135865c50f.JPG)

This is what my WA looks like after installing masque. If i disable masque it looks normal. Every single WA is very dark.
Masque has almost no options so going over them only took seconds. What's the issue here?

**Comment by StormFX** [comment, marked as answer]
Looks to me like Masque is being loaded after WeakAuras, but I could be wrong. Masque uses the default button templates so your first step should be to ask the developers of WeakAuras, as it's their responsibility to make sure WA is compatible with Masque. If there is an issue with Masque, they'll let me know.
